data/shared/citizen/scripting/v8/main.js

```diff
@@ -398,6 +398,8 @@ const EXT_LOCALFUNCREF = 11;
 				}
 
 				global.source = parseInt(source.substr(4));
+			} else if (IsDuplicityVersion() && source.startsWith('internal-net')) {
```

Comment:
Same memoization request here as above for Lua.

Reply:
Resolved above, will make the same change here of caching the result.